packages/core/core/src/utils/utils.js

```diff
@@ -0,0 +1,9 @@
+export function filesMatchingPattern(files) {
+  const globule = require('globule');
+  return globule.find(files, {nonull: true});
+}
+
+export function fileMatchesPattern(patterns, intendedPath) {
+  const globule = require('globule');
+  return globule.isMatch(patterns, intendedPath);
+}
```

Comment:
This was originally in `lib/utils/utils` but most of what was in there could either be inlined (as commented above) or removed entirely because of lack of usage inside Embark. This leaves us with a utils file with only two functions that are actually shared inside this package.